### PR TITLE
Timeline component

### PR DESCRIPTION
**Pull request by james-grant-moj**

### CODE REVIEW

File: src/modules/components/timeline/day-item/day-item.scss [hunk 1]
```diff
@@ -59,4 +59,10 @@
     color: $secondary-text-colour;
   }
 
+}
+
+gov-day-item:last-child() {
```
Comment (fabiotisci) [inline src/modules/components/timeline/day-item/day-item.scss]: we can't modify the sass files. We need to use the ones provided by Trevor so the solution needs to find a way to work with those

File: src/modules/components/timeline/timeline/timeline.scss [hunk 2]
```diff
@@ -7,6 +7,8 @@ $module: 'timeline';
   
   &-header {
     border-top: em(10) solid $govuk-blue;
+    padding-top: $gutter-half;
```
Comment (fabiotisci) [inline src/modules/components/timeline/timeline/timeline.scss]: Timeline must use Trevor's SCSS

Comment (james-grant-moj) [inline src/modules/components/timeline/timeline/timeline.scss]: This is Trevor's SCSS

File: src/modules/components/timeline/day-item/day-item.scss [hunk 3]
```diff
@@ -1,6 +1,3 @@
-
```
Comment (james-grant-moj) [inline src/modules/components/timeline/day-item/day-item.scss]: @fabiotisci I have double checked this SCSS file and the extra style has been removed. The latest file changes do not have any extra CSS unless it is from Trevor's PDK.

**Pull request merged**